### ISSUE TITLE
Anagramming ourself

### ISSUE THREAD

**Issue by lynn** [i7/counterfeit-monkey#208]
I ran into this sentence while facing Atlantida with Alex in charge of the body:

```
> shoot self with anagramming gun

The gun fires ruggedly into myself, but is unable to make anything
interesting out of a, a, a, d, e, l, n, r, and x. I recoalesce into
or original form.
```

That `or` is certainly a bug. When I put on the Britishising goggles, it's "`our` original form"; this might say something about the origin of the bug, but I don't understand Inform well enough to really say what's going on.

The relevant line of code is this one: https://github.com/i7/counterfeit-monkey/blob/9cb93921509f5e6cd185d845c4374f8c596bf654/Counterfeit%20Monkey.materials/Extensions/Counterfeit%20Monkey/Tools.i7x#L1811

And I guess it's interacting with this somehow: https://github.com/i7/counterfeit-monkey/blob/9cb93921509f5e6cd185d845c4374f8c596bf654/Counterfeit%20Monkey.materials/Extensions/Counterfeit%20Monkey/Tools.i7x#L1982-L1986

Anyhow, maybe such a drastic action needs special-casing entirely: "The gun fires ruggedly into myself" sounds passive, and we seem unpertured by shooting ourself into alphabet soup and then recoalescing. Maybe it's fine to reject the action with some witty response about how it seems like a bad idea.

**Comment by angstsmurf** [comment]
Thanks for reporting this!

This seems like a combination of several Inform bugs. There used to be many places were "[our]" was used with the expectation that "my" or "your" would be printed depending on the viewpoint, which of course conflicted with the "To say our" rule. Instead we just went with the unadapted "our", which kind of makes sense most of the time.

It certainly seems like a bug that it would affect [their] as well, and also that it seems to be attempting to say "our" rather than the correct "my".

**Comment by angstsmurf** [comment]
I tried to fix this with 85f0c47. It will now say:

```
>shoot self with gun
There is a momentary splitting sensation as the gun tries to make
something interesting out of a, a, a, d, e, l, n, r, and x. You
reacoalesce into our original form.
```
